mu4/appshell/appshell.qrc

```diff
@@ -19,5 +19,6 @@
         <file>qml/HomePage/AddonsContent.qml</file>
         <file>qml/DevTools/Audio/SynthSettings.qml</file>
         <file>qml/DevTools/Audio/SoundFontsPanel.qml</file>
+        <file>qml/DevTools/Audio/MIdiPorts.qml</file>
```

Comment:
the filename has mistake M**i**diPorts.qml

Reply:
fixed